### PR TITLE
Publish also arm64 architecture

### PR DESCRIPTION
**Pull request by javierdelapuente**
Applicable spec: <link>

### Overview

We want to build and publish an arm64 charm for the `nginx-integrator`.

For the integration tests to work, an arm64 version of `any-charm` is needed. Added in https://github.com/canonical/any-charm/pull/23

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

**Review by amandahla — APPROVED** [review]

File: .github/workflows/publish_charm_arm64.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Publish to edge arm64
+
+on:
+  push:
+    branches:
+      - main
+      - track/*
+
+jobs:
```
Comment (arturo-seijas) [inline .github/workflows/publish_charm_arm64.yaml:9]: Don't you need to specify the track?

File: .github/workflows/integration_test_arm64.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+jobs:
+  integration-tests-arm64:
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    secrets: inherit
+    with:
+      channel: 1.28-strict/stable
+      modules: '["test_cert_relation", "test_ingress_relation", "test_nginx_route"]'
+      juju-channel: 3.1/stable
+      self-hosted-runner: true
+      self-hosted-runner-arch: arm64
```
Comment (cbartz) [inline .github/workflows/integration_test_arm64.yaml:15]: Can you also specify the edge label (or check if its used per default in operator workflows)? We need people to test arm edge runners (and in general specifying all labels is encouraged, Otherwise an arbitrary size is used (medium, large, xlarge).

File: .github/workflows/publish_charm_arm64.yaml (hunk 1 above)
Comment (javierdelapuente) [reply .github/workflows/publish_charm_arm64.yaml:9]: The publish action is equal to the default for `amd64` (except that it uses the artifact from the arm64 integration test), by default it uses the `latest/stable`.

**Review by cbartz — APPROVED** [review]

**Review by jdkandersson — APPROVED** [review]
